dbt/models/iasworld/schema/iasworld.dweldat.yml

```diff
@@ -636,26 +688,32 @@ sources:
               - not_null:
                   name: iasworld_dweldat_sfla_not_null
                   config: *unique-conditions
-              - dbt_utils.accepted_range:
+              - accepted_range:
                   name: iasworld_dweldat_sfla_sf_between_1_and_10000
                   min_value: 1
                   max_value: 10000
+                  select_columns: *select-columns
                   config:
                     where: |
                       CAST(taxyr AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
                       AND class NOT IN ('208', '209', '211', '212', '236', '297')
                       AND cur = 'Y'
                       AND deactivat IS NULL
-              - dbt_utils.accepted_range:
+                  meta:
+                    description: sfla should be between 1 and 10000
+              - accepted_range:
                   name: iasworld_dweldat_sfla_mf_between_1_and_20000
                   min_value: 1
                   max_value: 20000
+                  select_columns: *select-columns
                   config:
                     where: |
                       CAST(taxyr AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
                       AND class IN ('208', '209', '211', '212')
                       AND cur = 'Y'
                       AND deactivat IS NULL
+                  meta:
+                    description: sfla should be between 1 and 20000
```

Comment:
```suggestion
                    description: sfla (bldg. square feet) should be between 1 and 20000 for multi-family and large single-family
```

Reply:
Fixed in 7ab148ef5729656b368e5e35515ce3a6513a358d.